R/read-zrxp.R

```diff
@@ -65,8 +65,8 @@ merge_meta_data <- function(ele) {
 #' @inheritParams tsdbr::ts_create_db
 #' @export
 ts_read_zrxp <- function(file = "tscbh.zrxp", utc_offset = -8L) {
-  check_string(file)
-  check_scalar(utc_offset, c(-12L, 14L))
+  chk_string(file)
+  chk_scalar(utc_offset, c(-12L, 14L))
```

Comment:
`chk_scalar` only checks if scalar need a separate `check_values()` for integer and range